byte TEST_RUNNING = 0
byte TEST_WARNING = 1
byte TEST_ERROR = 2
byte TEST_STALE = 3

byte test_ok
string message